src/css/components/activity_log.scss

```diff
@@ -0,0 +1,52 @@
+
+.activity-log {
```

Comment:
It should technically be `activity_log` for the base part of the classname, because the Activity Log is one thing. The `-` means that it's a separate part, such as an element or modifier.